server/src/main/java/org/elasticsearch/index/engine/ThreadPoolMergeExecutorService.java

```diff
@@ -230,10 +230,10 @@ private static long newTargetIORateBytesPerSec(
             );
         } else if (currentlySubmittedIOThrottledMergeTasks > concurrentMergesCeilLimitForThrottling
             && currentTargetIORateBytesPerSec < MAX_IO_RATE.getBytes()) {
-                // increase target IO rate by 10% (capped)
+                // increase target IO rate by 20% (capped)
                 newTargetIORateBytesPerSec = Math.min(
                     MAX_IO_RATE.getBytes(),
-                    currentTargetIORateBytesPerSec + currentTargetIORateBytesPerSec / 10L
+                    currentTargetIORateBytesPerSec + currentTargetIORateBytesPerSec / 20L
```

Comment:
I think what you did only adds 5%, it should be:

```suggestion
                    currentTargetIORateBytesPerSec + currentTargetIORateBytesPerSec / 5L
```

?